02 Writing Algorithms/30 Machine Learning/03 Popular Libraries/02 Tensorflow/06 Predict Labels.html

```diff
@@ -0,0 +1,14 @@
+<p>To predict the labels of new data, in the <code>OnData</code> method, get the most recent set of features and then call the <code>run</code> method with new features.</p>
+<div class="section-example-container">
+    <pre class="python">new_features, __ = self.get_features_and_labels()
+prediction = self.model.run(self.output, feed_dict={self.X: new_features})
+prediction = float(prediction.flatten()[-1])</pre>
+</div>
+
+<p>You can use the label prediction to place orders.</p>
+<div class="section-example-container">
+    <pre class="python">if prediction > 0:
```

Comment:
If the label (`prediction`) is the next closing price, shouldn't we check if it's greater than the current price, not if it's greater than 0?